utilities.py

```diff
@@ -7,7 +8,10 @@
 def getTileCoordinates(y, x):
     return (y + TILE_SIZE // 2) // TILE_SIZE, (x + TILE_SIZE // 2) // TILE_SIZE
 
-
+def generateSeed(n):
+    start = 10 ** (n - 1)
+    end = (10 ** n) - 1
+    return random.randint(start, end)
```

Comment:
10 ^ 60 is a huge number, isn't it? Do we still need this `MAX_NUMBER_OF_CRATES`?

Reply:
In fact we don't, already changed that

Reply:
do we even need to pass `n`?

Reply:
we don't, already changed that